src/ceph-disk

```diff
@@ -1593,7 +1603,10 @@ def prepare_dev(
         except subprocess.CalledProcessError as e:
             raise Error(e)
 
-        rawdev = get_partition_dev(data, 1)
```

Comment:
Nevermind. 

The ```update_partition``` above means partprobe is called and the kernel has fired the events related to the partition that was just created. If it was not the case the get_partition_dev could not succeed because the partition would not exist in /dev/vdb1 or whatever. The udev event that creates the partuuid is also guaranteed to be complete because of the udevadm settle that is part of update_partition.